src/main/java/org/spongepowered/common/mixin/core/item/MixinEnchantment.java

```diff
@@ -108,13 +109,12 @@ public boolean canBeAppliedByTable(ItemStack stack) {
     }
 
     @Override
-    public boolean isCompatibleWith(Enchantment ench) {
+    public boolean isCompatibleWith(EnchantmentType ench) {
         return canApplyTogether((net.minecraft.enchantment.Enchantment) ench);
     }
 
-    @Intrinsic
     public String enchantment$getName() {
-        return shadow$getName();
+        return I18n.translateToLocal(shadow$getName());
```

Comment:
Is this change intended?

Reply:
Yeah as the getName method previously returned the translation key of "enchantment." + name. For example "enchantment.waterWalker" or similar.